### PR TITLE
changed ui in complain.php

### LINKED ISSUE #9: Enhance the ui of complain page

### PR DESCRIPTION
**Pull request by Ankit7Das**
fix #9 
p

### CODE REVIEW

File: complain.php [hunk 1]
```diff
@@ -1,10 +1,5 @@
 <?php 
 	include("./functions/init.php");
-	if(logged_in() == false){
```
Comment (atm1504) [inline complain.php]: Revert this changes. Its the most important part of the file.

File: complain.php [hunk 2]
```diff
@@ -36,111 +31,234 @@
 	<link rel="stylesheet" type="text/css" href="css/util.css">
 	<link rel="stylesheet" type="text/css" href="css/main_complain.css">
 <!--===============================================================================================-->
+	<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"/>
 </head>
+
+<style>
```
Comment (atm1504) [inline complain.php]: Write all the css part in main_complain.css

File: complain.php [hunk 3]
```diff
@@ -36,111 +31,234 @@
...
-				<div class="wrap-input100 validate-input bg1" data-validate="Please Type Your Name">
-					<span class="label-input100">FULL NAME *</span>
-					<input class="input100" type="text" name="name" id="name" placeholder="Enter Your Name">
-				</div>
-
-				<div class="wrap-input100 validate-input bg1 rs1-wrap-input100" data-validate = "Enter Your Email (e@a.x)">
-					<span class="label-input100">Email *</span>
-					<input class="input100" type="text" name="email" id="email" placeholder="Enter Your Email ">
-				</div>
+				<form>
```
Comment (atm1504) [inline complain.php]: Don't change the value under name attribute. Doing so will completely crash the page and disable all the functionalities. Keep the value of name and id tag as it is.
Don't remove the hidden input fields in the forms. They are necessary.

All this things are necesaary to process the data in the backend.

**Comment by Ankit7Das** [comment]
@atm1504 bhaiya please do review it once
